src/components/charts/colorMixingTool/index.jsx

```diff
@@ -1,76 +1,125 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import { SVGIcon, SelectField } from 'react-md';
 import Button from '../../site/button/index.js';
+// import arrowDropDown from 'icons/arrow_drop_down.svg';
```

Comment:
I see that you're trying to load in an icon. Those can be found and imported from the `/site/icons/` folder. You just need to import the component like: `import ArrowDown from '../../site/icons/ArrowDown';` and then set it in the component like:

```
<SelectField
  ...
  dropdownIcon={<ArrowDown />}
  ... />
```
the spot in the docs is [here](https://react-md.dev/v1/components/select-fields?tab=1#select-field-proptypes-dropdown-icon)

To get the standard styling to work, you'll need to import the @mixin for the select-field like this:

```@include react-md-select-fields;```

found on this page: [react-md-select-fields](https://react-md.dev/v1/components/select-fields?tab=2#mixin-react-md-select-fields)